compsequential_test.go

```diff
@@ -12,6 +12,34 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestNoResultsSequential(t *testing.T) {
+	require := require.New(t)
+	rs := []*SequentialRouter{
+		{
+			Timeout:     time.Second,
+			IgnoreError: true,
+			Router:      Null{},
+		},
+		{
+			Timeout:     time.Second,
+			IgnoreError: true,
+			Router: &Compose{
+				ValueStore:     newDummyValueStore(t, []string{"a"}, []string{"av"}),
+				PeerRouting:    Null{},
+				ContentRouting: Null{},
+			},
+		},
+	}
+
+	cs := NewComposableSequential(rs)
+
+	v, err := cs.GetValue(context.Background(), "a")
+	require.NoError(err)
+	require.Equal("av", string(v))
+
+	require.Equal(2, len(cs.Routers()))
```

Comment:
nit: we need the same test for Parallel (even more important, because thats what we use in Kubo)

Reply:
the test for parallel is here: https://github.com/libp2p/go-libp2p-routing-helpers/pull/68/files#diff-a6a74be66271405e953520b0ae14eeaa2dde6ae37d599d21f7b865927935882fR40